test/utils/helper.ts

```diff
@@ -0,0 +1,165 @@
+import fs, { promises as fsp } from 'fs'
+import path from 'path'
+import os from 'os'
+import { fork } from 'child_process'
+import * as debug from './debug'
+
+export function stripANSIColor(str: string) {
+  return str.replace(
+    /[\u001b\u009b][[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-ORZcf-nqry=><]/g,
+    '',
+  )
+}
+
+export async function existsFile(filePath: string) {
+  try {
+    await fsp.access(filePath)
+    return true
+  } catch (err: any) {
+    if (err.code === 'ENOENT') {
+      return false
+    }
+    throw err
+  }
+}
+
+export function assertContainFiles(dir: string, filePaths: string[]) {
+  const results = []
+  for (const filePath of filePaths) {
+    const fullPath = path.resolve(dir, filePath)
+    const existed = fs.existsSync(fullPath)
+    if (existed) {
+      results.push(filePath)
+    }
+  }
+  expect(results).toEqual(filePaths)
+}
+
+export async function assertFilesContent(
+  dir: string,
+  contentsRegex: Record<string, RegExp | string>,
+) {
+  const distFiles = Object.keys(contentsRegex)
+  for (const relativeFile of distFiles) {
+    const file = path.join(dir, relativeFile)
+    expect({
+      [file]: (await existsFile(file)) ? 'existed' : 'missing',
+    }).toMatchObject({ [file]: 'existed' })
+  }
+
+  for (const [file, regex] of Object.entries(contentsRegex)) {
+    const content = await fsp.readFile(path.join(dir, file), {
+      encoding: 'utf-8',
+    })
+    if (regex instanceof RegExp) {
+      expect(content).toMatch(regex)
+    } else {
+      expect(content).toContain(regex)
+    }
+  }
+}
+
+// bundle.min.js => .min.js
+export const fullExtension = (filename: string) =>
+  filename.slice(filename.indexOf('.'))
+
+export function getChunkFileNamesFromLog(log: string) {
+  return log.split('\n').map((line: string) => {
+    return line.replace(/- \d+ K?B/, '').trim()
+  })
+}
+
+export async function deleteFile(f: string) {
+  if (await existsFile(f)) {
+    await fsp.unlink(f)
+  }
+}
+
+export async function createTempDir(): Promise<string> {
+  const tempDir = os.tmpdir()
+  const tempDirPrefix = 'bunchee-test'
+  const tempDirPath = path.join(tempDir, tempDirPrefix)
+
+  return await fsp.mkdtemp(tempDirPath)
+}
+export async function removeDirectory(tempDirPath: string) {
+  debug.log(`Clean up ${tempDirPath}`)
+  await fsp.rm(tempDirPath, { recursive: true, force: true })
+}
+
+type ExcuteBuncheeResult = {
+  code: number
+  stdout: string
+  stderr: string
+}
+
+export async function executeBunchee(
```

Comment:
Let's move this to test/utils/cli. along with createCliTest.

Reply:
Other looks pretty awesome as well, is this still a draft or it's ready to review?

Reply:
There are still some test cases that have not been modified. I'll complete these today.

Reply:
> Let's move this to test/utils/cli. along with createCliTest.

I think `executeBuchee` in cli tests is same as `executeBunchee`, so I define it in helper.